model_manager/src/transformers/BaseTransformer.py

```diff
@@ -0,0 +1,26 @@
+from abc import ABC, abstractmethod
+
+class BaseTransformer:
+    @abstractmethod
+    def __init__(self, config):
+        """
+        config: dict passed from the pv_mappings.yaml files
+        """
+        pass
+    
+    @abstractmethod
+    def transform(self):
+        """
+        Initiate transform function to transform the input data
+        """
+        pass
+    
+    @abstractmethod
```

Comment:
Type hinting and description of what the objects are:
* pv_name: is it a list of PVs? if so, what kind of object, is it string?
* value: what is meant by values here?